packages/cra-template-typescript/template/src/hooks/useFelaEnhanced.ts

```diff
@@ -1,18 +1,34 @@
 import React from 'react';
-import { combineMultiRules } from 'fela-tools';
-import { useFela } from 'react-fela';
+import { combineMultiRules, TNormalizedMultiRule } from 'fela-tools';
+import { TRule } from 'fela';
+import { useFela, Rules as ReactFelaRules } from 'react-fela';
+
+import { Theme } from 'styles/theme';
```

Comment:
Is `Theme` type really exported from there?

Reply:
good point, will check

Reply:
Add `Theme` type